events/2019-08-29-ng-dinner.yml

```
@@ -0,0 +1,5 @@
+name: ngУжин
+date: 29.08.2019
+time: 20:00-NaN
```

Добавь, пожалуйста, время окончания
```suggestion
time: 20:00-23:59
```

---

done